charts/generic-service/templates/ingress.yaml

```diff
@@ -15,18 +14,14 @@ metadata:
   {{- if .Values.allowlist }}
     nginx.ingress.kubernetes.io/whitelist-source-range: {{ include "app.joinListWithComma" .Values.allowlist | quote }}
   {{- end }}
-    external-dns.alpha.kubernetes.io/set-identifier: {{ $ingressName }}-{{ .Release.Namespace }}-green
-  {{- if not .Values.ingress.v1_2_enabled }}
+    external-dns.alpha.kubernetes.io/set-identifier: {{ $fullName }}-{{ .Release.Namespace }}-green
+    external-dns.alpha.kubernetes.io/aws-weight: "0"
```

Comment:
weighting moved back here so can override to 0 in this file and 100 in the new ingress

Reply:
If people keep the default then they won't get the new ingress - what happens with this weight? Is it ignored?

Reply:
the CP user guide says that if the _only_ ingress has a weight of 0 it still gets all the traffic

Reply:
indeed.  From https://user-guide.cloud-platform.service.justice.gov.uk/documentation/other-topics/Switch-ingress-to-v1-ingress-controller.html#step-1-changes-to-your-current-ingress-resource:
> You must set external-dns.alpha.kubernetes.io/aws-weight value to “0” for the current ingress. This does not mean traffic will stop, 0 is a valid value for a single ingress.